_includes/footer-navigation.html

```diff
@@ -1,13 +1,34 @@
-<div class="footer-navigation grid-container">
-  <div class="footer-navigation__section margin-y-2">
+<div class="footer-navigation grid-container grid-row">
+  <div class="footer-navigation__section grid-col-12 margin-y-2 tablet:grid-col-auto">
     <img
       src="{{ site.baseurl }}/assets/img/logo.svg"
       alt="Login.gov logo"
       width="191"
       height="27"
     />
   </div>
-  <div class="footer-navigation__section">
+  <div class="grid-col-fill"></div>
+  <div class="grid-col-12 tablet:grid-col-auto">
+    <nav
+      aria-label="Footer navigation"
+      class="grid-row grid-gap-2 tablet:grid-gap-6"
+    >
+      {% assign login_base = 'https://login.gov' %}
```

Comment:
Minor: While technically both versions work, the [configured canonical domain should be `www.`](https://github.com/GSA-TTS/identity-site/blob/a2a42a1d00d1d58dc5c45c5efaa4cbf7aba00898/_config.yml#L4), and ideally we'd have redirects set up to force this, where we'd want to avoid an unnecessary redirect hop ([LG-10513](https://cm-jira.usa.gov/browse/LG-10513)). 

```suggestion
      {% assign login_base = 'https://www.login.gov' %}
```